docs/pages/setup/reference/predicate-language.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: Predicate Language
+description: How to use predicate language used to define filter conditions.
+h1: Predicate Language
+---
+
+Predicate language is used to define filter or where condition for select resources.
+It is also used as a query language to filter and search through a [list of select resources](#resource-filtering).
+
+The predicate language support slightly differs depending on where it is used as explained in the next sections.
```

Comment:
```suggestion
The predicate language uses a slightly different syntax depending on whether it is used in:

- [Role resources](#scoping-allow-deny-rules-in-role-resources)
- [Resource filtering](#resource-filtering)
```

I thought it would make sense to add a mini-TOC here so a reader can find the relevant section.